README.md

```diff
@@ -1,40 +1,35 @@
-# Python Client Library
+# Library `odm_sdk`
 
 ### Documentation
 
    [stable](http://genestack-client.readthedocs.io/en/stable/)
 
 ### Installation
 
-   - install library and scripts:
-     ```
-     pip install genestack-client
-     ```
-     if you need the latest development version:
-     ```
-     pip install --upgrade https://github.com/genestack/python-client/archive/master.zip
-     ```
-   - setup default user
-     ```
-     genestack-user-setup init
-     ```
-
-### Repo
-
-  ```
-  docs/                             - folder with documentation sources
-  genestack_client/                 - folder with library
-  tests/                            - developer tests, require pytest to run.
-  genestack-user-setup              - script to setup user (installed as script)
-  genestack-shell                   - shell (installed as script)
-  setup.py                          - installation setup for genestack_client library
-  ```
+```shell
+python3 -m pip install odm-sdk
```

Comment:
I could not find good examples, so I am not sure: how to name the package? Currently it's odm-sdk for installation and odm_sdk in python code. Shouldn't we keep just one name? 

I found this example: https://github.com/googleapis/google-api-python-client. It's `google-api-python-client` for installation and `googleapiclient` inside python code

Reply:
> Shouldn't we keep just one name?

It's the same name just in a different case because of python limitations, python module can't be in kebab-case. I've seen this approach in a lot of projects. e.g.
- https://github.com/banglafighter/pweb-cli
- https://github.com/cleanlab/cleanlab-studio
- https://github.com/microsoft/playwright-pytest
- https://github.com/allure-framework/allure-python
- etc.

For me this list could continue infinitely, it's harder to find packages that do not follow this approach rather than vice-versa.

Reply:
- pweb-cli - a very small project, I don't think we can count it as a valid reference.
- cleanlab-studio - better and indeed, follows the same pattern, but still not really well-known.
- playwright - not valid, it's just playwright in python. 
- allure - not valid, it's just allure in python.

In general I am not against this approach, I just want to collect a little bit more evidence

Reply:
Even our generated `odm-api` follows this rule...
A package called `odm-api`, but a python module inside of the package called `odm_api`.

Reply:
For sure, helpful link here:
https://pypi.org/search/?q=cli&page=1